doc/man3/BIO_s_mem.pod

```diff
@@ -114,10 +135,6 @@ preceding that write operation cannot be undone.
 Calling BIO_get_mem_ptr() prior to a BIO_reset() call with
 BIO_FLAGS_NONCLEAR_RST set has the same effect as a write operation.
 
-=head1 BUGS
-
-There should be an option to set the maximum size of a memory BIO.
-
```

Comment:
I don't see why you want to remove this.  Yes, you added an initial dgram size, but that's not the same thing!

Reply:
Because it is in the bugs section, but isn't a bug. Its a feature request. We don't put feature requests in the documentation - we put them in github. But I didn't particularly feel inclined to create this in github either (no one has so far actually wanted this feature - or at least no one has been bothered to implement it).